.github/workflows/stale-issue.yml

```diff
@@ -91,3 +91,31 @@ jobs:
             If you are still interested in contributing this, please ensure that
             it is rebased against the latest branch (usually `master`), all review
             comments have been addressed and the build is passing.
+  close-pending-reproducer:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v6
+        with:
+          # GLOBAL ------------------------------------------------------------
+          operations-per-run: 30
+          exempt-all-milestones: true
+          days-before-stale: -1
+          only-labels: 'pending:reproducer'
+
+          # ISSUES ------------------------------------------------------------
+          stale-issue-label: 'pending:reproducer'
+          close-issue-label: 'closed:unreproducible'
```

Comment:
It looks like you are missing the `days-before-issue-close` configuration, no?
Even if the default is 7 days, let's be explicit about it.

Reply:
Wouldn't it be `days-before-close` to apply both for issues and PRs?